test/plugins/features/movies/controller.test.js

```diff
@@ -23,14 +43,67 @@ describe('movie controller', () => {
 
   });
 
-  describe('get', () => {
+  describe('list', () => {
+
+    it('list movies released after year_start', () => {
+      const payload = { year_start: 1983 };
+
+      return Controller.list(payload)
+      .then((json) => {
+        json.map((movie) => {
+          expect(movie.attributes.release_year).to.be.at.least(1983);
+        });
```

Comment:
It's good that you're iterating and making sure that all of the returned models satisfy the filter, but there's a problem if there's a bug in your code that makes the returned collection have 0 models. In that case, calling `.map` on an empty iterable (e.g. array, collection, etc), won't call the mapping function even once. So having a test that does the following will always pass:

```js
[].map(() => {
  throw new Error('errrr');
});
```

To solve this problem, in addition to asserting the contents, you should also assert the length of the iterable is what you expect. In this case, I _think_ you can do `expect(movies).to.have.length(2);`.